packages/relay/src/lib/eth.ts

```diff
@@ -1643,9 +1643,13 @@ export class EthImpl implements Eth {
 
     const selector = getFunctionSelector(call.data!);
 
+    // The force-to-consensus-by-selector logic is active only when process.env.ETH_CALL_FORCE_TO_CONSENSUS_BY_SELECTOR is set to 'true'.
+    // When eth_call is invoked with a selector listed in specialSelectors, it will be routed through the consensus node, regardless of ETH_CALL_DEFAULT_TO_CONSENSUS_NODE.
+    // note: this feature is a workaround for when a feature is supported by consensus node but not yet by mirror node.
+    // Follow this ticket https://github.com/hashgraph/hedera-json-rpc-relay/issues/2984 to revisit and remove special selectors.
+    const specialSelectors = process.env.ETH_CALL_CONSENSUS_SELECTORS || [''];
     const shouldForceToConsensus =
-      process.env.ETH_CALL_FORCE_TO_CONSENSUS_BY_SELECTOR == 'true' &&
-      constants.ETH_CALL_SELECTORS_ALWAYS_TO_CONSENSUS.indexOf(selector) !== -1;
+      process.env.ETH_CALL_FORCE_TO_CONSENSUS_BY_SELECTOR === 'true' && specialSelectors.includes(selector);
```

Comment:
Just a thought, but this can be simplified by removing `ETH_CALL_FORCE_TO_CONSENSUS_BY_SELECTOR` altogether. When the functionallity needs to be disabled just leave `ETH_CALL_CONSENSUS_SELECTORS` as an empty array. This way there is 1 less env variable to maintain

Reply:
yes fair point!

Reply:
updated with the removal of `ETH_CALL_FORCE_TO_CONSENSUS_BY_SELECTOR`